examples/shaders/fade-out/fade-out.html

```diff
@@ -0,0 +1,152 @@
+<!--
+ If not stated otherwise in this file or this component's LICENSE file the
+ following copyright and licenses apply:
+
+ Copyright 2020 RDK Management
+
+ Licensed under the Apache License, Version 2.0 (the License);
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" />
+    <style>
+        body{
+            margin: 0;
+            padding: 0;
+        }
+    </style>
+</head>
+<body>
+<script type="module">
+    import lng from '../../../src/lightning.mjs';
+
+    class FadeOutExample extends lng.Application {
+        static _template() {
+            return {
+                w: 1920, h: 1080,
+                FadeOut1: {
+                    x: 90,
```

Comment:
Examples could be improved using `flex` and variables like `const SRC='...'; const SIZE = 300`. Also, it's better to use local images rather than external ones.

Reply:
added the const for SRC and SIZE. But won't incorporate flex as I don't want to overcomplicate the examples with that functionality(hit me up in chat for more info).